Content.Client/Stylesheets/StyleNano.cs

```diff
@@ -1423,6 +1423,10 @@ public StyleNano(IResourceCache resCache) : base(resCache)
                     .Prop(TextureRect.StylePropertyTexture, resCache.GetTexture("/Textures/Interface/Nano/ntlogo.svg.png"))
                     .Prop(Control.StylePropertyModulateSelf, Color.FromHex("#757575")),
 
+                Element<TextureRect>().Class("PirateLogoDark") // Frontier
+                    .Prop(TextureRect.StylePropertyTexture, resCache.GetTexture("/Textures/Interface/Nano/piratelogo.svg.png")) // Frontier
+                    .Prop(Control.StylePropertyModulateSelf, Color.FromHex("#757575")), // Frontier
```

Comment:
Is this going to conflict with the reworked stylenano from upstream?

Reply:
If you're referring to #29903, _yup_.  I'll need to give it some thought.

Reply:
Yolo till update?

Reply:
Could, it's a marginal change anyways, just a little skull icon in the corner.

Reply:
U+1F480 Moment